src/spatial/models/site_category_model.py

```
@@ -137,133 +338,162 @@ def fallback_name():
+
+        for way in result.ways:
+            tags = way.tags
+            landuse = tags.get("landuse", "unknown")
+            natural = tags.get("natural", "unknown")
+            waterway = tags.get("waterway", "unknown")
+            highway = tags.get("highway", "unknown")
+            center_lat = way.center_lat
+            center_lon = way.center_lon
+            area_name = tags.get("name", "Unnamed")
+
+            debug_info.append(f"Found OSM data:")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)